xla/service/gpu/transforms/cancel_all_gather_dynamic_slice.cc

```diff
@@ -0,0 +1,82 @@
+/* Copyright 2024 The OpenXLA Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+#include "xla/service/gpu/transforms/cancel_all_gather_dynamic_slice.h"
```

Comment:
I'd add new line btw === and include. Other cc/h files in xla codebase have it.

I also noticed that imports are not sorted

Reply:
Added new line. In other XLA headers the pass name header is the first include, then new line, then rest of includes. The rest of my includes are sorted.